krux_redis/cli.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# © 2017 Krux, A Salesforce Company
+#
+
+from __future__ import absolute_import
+
+from krux.cli import get_group
+
+
+DEFAULT_TIMEOUT = 0.5   # in seconds
+
+
+# Designed to be called from krux.cli, or programs inheriting from it
+def add_redis_cli_arguments(parser):
```

Comment:
Current practice is to put this under `client.py` and build a shallow CLI under `cli.py`. Feel free to disagree since it is only there because it has been there before.